### ISSUE TITLE
What are the ouput files formats I can create? 

### ISSUE THREAD

**Issue by maniin** [nasa/NASTRAN-95#7]

**Comment by AeroDME** [comment]
The output format is output stream 6 (STDOUT) which is the f06 output the MSC  users will recognize.

OP2 & XDB formats are not supported that I know of.

**Comment by maniin** [comment]
Thanks Dan,
When I tried PARAM POST -1 , it throws error.
by default it gives .out file, which is equivalent to f06(Is my understanding correct?).
Is there any other binary format where I can read programatically?

**Comment by AeroDME** [comment]
I dont think so. Unless there is a fork of the source which has that feature. I'm not sure when OP2 came around, but I am pretty sure it was an MSC thing.

**Comment by SteveDoyle2** [comment]
You can output to OP2, but as far as I know, an alter is required.  Something like:

    ALTER      110
    OUTPUT2    OUGV1,OEF1,OES1,,//-1/12////*MSC*  $
    OUTPUT2    ,,,,, //-9/12/  $
    ENDALTER
